docs/user-guide/custom-themes.md

```diff
@@ -148,32 +150,39 @@ folder on all pages you would do this:
 ```
 
 #### homepage_url
+
 Provides a relative path to the documentation homepage.
 
 #### mkdocs_version
+
 Contains the current MkDocs version.
 
 #### build_date_utc
+
 A Python datetime object that represents the date and time the documentation
 was built in UTC. This is useful for showing how recently the documentation
 was updated.
 
-
 ### Page Context
+
 The page context includes all of the above Global context and the following
```

Comment:
Someone has a bad habit of adding an extra blank line before headers (and occasionally not including one after). If this is you, you should probably stop. ;-)